requirements/edx/github.txt

```diff
@@ -96,7 +96,7 @@ git+https://github.com/edx/edx-proctoring.git@0.17.0#egg=edx-proctoring==0.17.0
 -e git+https://github.com/mitodl/edx-sga@172a90fd2738f8142c10478356b2d9ed3e55334a#egg=edx-sga
 -e git+https://github.com/open-craft/xblock-poll@v1.2.3#egg=xblock-poll==1.2.3
 git+https://github.com/edx-solutions/xblock-drag-and-drop-v2@v2.0.15#egg=xblock-drag-and-drop-v2==2.0.15
-git+https://github.com/raccoongang/xblock-pdf.git@v1.0.2-rg#egg=xblock-pdf
+git+https://github.com/barik94/xblock-pdf@1.0.1-ifnmu#egg=xblock-pdf
```

Comment:
Why do you use this xblock form barik94 repository instead of raccoongang? 
I think we need to set this version to the raccoongang repository and use it.

Reply:
Agree with Andrey personal repos are not an appropriate solution.